app.js

```diff
@@ -11,6 +11,8 @@ var indexRouter = require('./routes/index');
 var favoriteSongsRouter = require('./routes/api/v1/favoriteSongs');
 
 var app = express();
+// app.set("port", process.env.PORT || 3000);
+app.locals.title = "Play Play";
 
```

Comment:
@tschaffer1618 see if putting the dev script in the package.json fixes this problem